lib/modules/manager/copier/extract.spec.ts

```diff
@@ -21,6 +21,25 @@ describe('modules/manager/copier/extract', () => {
       });
     });
 
+    it('extracts repository and version from .copier-answers.yml with ssh URL', () => {
+      const content = `
+        _commit: v1.0.0
+        _src_path: git@github.com:renovatebot/somedir/renovate.git
+      `;
+      const result = extractPackageFile(content);
+      expect(result).toEqual({
+        deps: [
+          {
+            depName: 'git@github.com:renovatebot/somedir/renovate.git',
```

Comment:
I think this can be messaged further to use GitHub tags datasource.
we do such things in other managers already.
can of cause be done in a separate PR